features/steps/consistency.py

```diff
@@ -40,28 +40,28 @@ def step_impl(context, image, file_type):
 def step_impl(context, image, file_type):
     found_executable = get_dist_name()
     file_flag = ''.join(('-', file_type.lower()))  # being lazy here, letting file type match the switch
-    args = [found_executable, file_flag, "-compress", image]
+    args = [found_executable, file_flag, "-no-denoise", "-color", "none", "-compress", "-no-crop", image]
     run_conversion(args)
 
 
 @when(u'the {image} is denoised and converted by the code')
 def step_impl(context, image):
     found_executable = get_dist_name()
-    args = [found_executable, '-dng', '-denoise', image]
+    args = [found_executable, '-dng', image]
     run_conversion(args)
 
 
 @when(u'the {image} is denoised and converted by the code to a cropped color TIFF')
 def step_impl(context, image):
     found_executable = get_dist_name()
-    args = [found_executable, '-tiff', '-denoise', '-color', 'AdobeRGB', '-crop', image]
+    args = [found_executable, '-tiff', '-color', 'AdobeRGB', image]
     run_conversion(args)
 
 
 @when(u'the {image} is converted to tiff {output_format}')
 def step_impl(context, image, output_format):
     found_executable = get_dist_name()
-    output_switch = '-crop'
+    output_switch = '-tiff' # totally harmless dummy
     if output_format == 'UNPROCESSED':
         output_switch = '-unprocessed'
     if output_format == 'QTOP':
```

Comment:
Nothing's wrong with elif.  Python really doesn't do switches, so if you think elif is appropriate, go for it.